BeeQuantAI/platform_app
# Feat: Added the new feature stay signed in

Added the new feature stay signed in:
- Added and used logout mutation in two logout topbar and sidebar component
- Added response interceptor in Apollo.ts to get new access token and x-auth-status and save into localstorage
- Changed the 'Remember me' to 'Stay signed in' at LoginFormGroup.tsx
- Changed the token validation logic, from token validation to auth status validation at auth.ts
- Optimized the getUserInfo method implement logic in userHooks.ts. it should not run when at login page
- Optimized the dependency of useEffect() in WrappedRoutes.tsx to implement the combined guard when url changed

Resolve CP-20

**Screenshot:** 
Jest:
<img width="1080" alt="image" src="https://github.com/user-attachments/assets/3f6edd67-5c78-48d4-88ee-f50eb12e15b4">

**Steps to test:**
1. Change const refreshToken = this.jwtService.sign({ id: user.id }, { expiresIn: expiresFreshToken }); to const refreshToken = this.jwtService.sign({ id: user.id }, { expiresIn: '1m' }); at the auth.service.ts from backend
2. Change else { accessToken = this.jwtService.sign({ id: user.id }); } at the auth.service.ts generateAccessToken function  from backend
3. Add getUserInfo method into any one component that to send a request, like in url http://localhost:5173/crypto/exchange
4. activate the request  at the page
5. Wait for 1 minute, and click button exchanges, and then click any other button like Prices at Sidebar or refresh the page to see the effect. it will push to login page.
